### PR TITLE
added pirate personality type and 12 pirate messages.

### LINKED ISSUE #199: Add new hydrate message personality type: Pirate
## Feature Reasoning

With the introduction of personality types for hydration break messages, it would be nice to add more new personality types. One such personality type to add is: `pirate`.

## Feature Description

The purpose of this feature request is to add a new personality type `pirate` to the currently existing personality types available for the hydrate break messages. The new personality type should be made selectable via the personality type dropdown menu in the hydrate reminder configuration menu.

To start off, adding at least 12 new messages should be good and more can be added later.
An example of a pirate message could be something like: `Arr, Matey! Ain't nothin' quite like a nice cup of water to hydrate with`

## Possible Alternatives

If not really pirate, any messages that could be considered seafaring or adventurous can be considered OK for this personality type.

## Additional Context

To complete this work, the following two files need to be edited:
- [HydrateReminderPersonalityType.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java)
- [HydrateBreakMessageDictionary.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java)

Looking into these two files should give a clear indication on how to implement this feature, but please do ask any questions in the comments below if needed. Please also add and/or update unit tests where necessary.

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[](https://discord.gg/RTgxfFW9mS)

### PR DESCRIPTION
**Pull request by Lumary2**
## Associated Issue

Closes #199 

## Testing Details

Operating System: 
RuneLite Version:

### CODE REVIEW

**Comment by jmakhack** [comment]
@Lumary2, looks like the build is failing after your changes with the following error messages:

```
Starting a Gradle Daemon (subsequent builds will be faster)

/Users/runner/work/hydrate-reminder/hydrate-reminder/src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java:15: error: invalid method declaration; return type required
> Task :compileJava FAILED
    PIRATE("Pirate");
    ^
/Users/runner/work/hydrate-reminder/hydrate-reminder/src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java:15: error: illegal start of type
    PIRATE("Pirate");
           ^
2 errors
```

Can you take a look into fixing this?

**Comment by Lumary2** [comment]
Fixed it, sorry about that!

File: src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
     POLITE("Polite"),
     MOTIVATIONAL("Motivational"),
     ROMANTIC("Romantic"),
+    PIRATE("Pirate");
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java:15]: Should be a comma instead of a semicolon here.

Comment (Lumary2) [reply src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java]: Ah again 🙈hope it works now😀

**Comment by jmakhack** [comment]
@all-contributors please add @Lumary2 for code

**Review by jmakhack — APPROVED** [review]
Looks good and everything is passing! Thanks for the contribution!

**Pull request merged**